core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.controller.ts

```diff
@@ -100,6 +142,68 @@ angular.module('oppia').controller('TranslationModalController', [
       $scope.activeWrittenTranslation.html = '';
     };
 
+    $scope.getElementTexts = function(elements, type) {
+      const textWrapperLength = 6;
+      const attributes = Array.from(elements, function(element: HTMLElement) {
+        if (element.localName === 'oppia-noninteractive-image') {
+          const attribute = element.attributes[type].value;
+          return attribute.substring(
+            textWrapperLength, attribute.length - textWrapperLength);
+        }
+      });
+      return attributes.filter(attribute => attribute);
+    };
+
+    $scope.copiedAllElements = function(originalElements, translatedElements) {
```

Comment:
This function name should be the opposite right? Since the return value is negated, currently this function returns:
* false if every element matches
* true if not every element matches

Maybe we shouldn't negate the return value here but negate it when you're declaring `hasUncopiedImgs`.

Reply:
Function name changed.